app/locale/en.coffee

```diff
@@ -2266,7 +2266,7 @@ module.exports = nativeDescription: "English", englishDescription: "English", tr
     view_project: "View Project"
     unpublished: "(unpublished)"
     view_arena_ladder: "View Arena Ladder"
-    resource_hub: "Resource Hub"
+    resource_hub: "Teacher Toolkit" # {change}
```

Comment:
The idea is that the copy-i18n-tags.js script will copy that # {change} tag to the locale files that already had it, so that anyone who is really paying close attention can translate changed tags. But the script also eventually clears the # {change} tags on the next run, so in practice I don't know how useful this is.